content/en/docs/tasks/manage-daemon/pods-some-nodes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Running Pods on Only Some Nodes
+content_type: task
+weight: 30
```

Comment:
How did we decide what number to use here?

Reply:
`weight` decides the relative ordering of these pages in the section's list. 'Performing a Rolling Update' was weighted at 10 while 'Performing a rollback' was weighted at 20 and hence they were ordered 1st and 2nd respectively. I think both of them should appear along with each other. So either I have to order it above them or below them. As it would be a new page I decided to order it below them and hence weighted it at 30